stream.js

```diff
@@ -83,6 +83,8 @@ function WebSocketStream(target, protocols, options) {
   socket.onerror = onerror
   socket.onmessage = onmessage
 
+  proxy.on('error', function(){})
```

Comment:
Ignore errors? It can lead to even more long debugging then you have had.

Reply:
It seems we have two options here: Ignore unhandled errors or have unhandled errors lead to crashes. My first pull request was to document how to handle errors so you don't have crashes and I was asked to instead fix the issue. I don't think there is a third option but I'd be very happy to learn something.